pkg/querier/http.go

```
@@ -31,19 +31,16 @@ type QueryHandlers struct {
 	upstream querierv1connect.QuerierServiceHandler
 }
 
-// LabelValuesHandler only returns the label values for the given label name.
+// LabelValues only returns the label values for the given label name.
 // This is mostly for fulfilling the pyroscope API and won't be used in the future.
-// /label-values?label=__name__
+// For example, /label-values?label=__name__ will return all the profile types.
 func (q *QueryHandlers) LabelValues(w http.ResponseWriter, req *http.Request) {
 	label := req.URL.Query().Get("label")
 	if label == "" {
 		http.Error(w, "label parameter is required", http.StatusBadRequest)
 		return
 	}
-	var (
-		res []string
-		err error
-	)
+	var res []string
 
 	if label == "__name__" {
 		response, err := q.upstream.ProfileTypes(req.Context(), connect.NewRequest(&querierv1.ProfileTypesRequest{}))
```

Something's wrong we're swallowing the error